### DISCUSSION TITLE
如何设定stencil的画布初试缩放率？

### DISCUSSION THREAD

**Discussion by nielinjie** [antvis/X6#3490] [Q&A]
我的节点比较大，希望在stencil里面放缩小版。我理解stencil里面也是graph渲染的，应该可以set它的缩放率，但不知道如何通过stencil实例拿到里面的graph实例。

**Comment by XueMeijing** [comment]
不用graph，可以看下[常见问题3](http://x6.antv.antgroup.com/tutorial/plugins/dnd#%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98)
